lib/ohai/plugins/solaris2/network.rb

```diff
@@ -101,7 +101,8 @@ def full_interface_name(iface, part_name, index)
     cint = nil
 
     so.stdout.lines do |line|
-      if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      # if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      if line =~ /^([0-9a-zA-Z\.\:\-]+): \S+ mtu (\d+) index (\d+)/
```

Comment:
I find linking to a rubular gisst to be really helpful with ohai regex usage. It's really hard to tell what these are attempting to detect without one. Here's an example: https://github.com/chef/ohai/blob/484d8b538d547096ed4472ca5a795d21d0d902db/lib/ohai/plugins/zpools.rb#L69